lib/resource-mapper.js

```diff
@@ -78,7 +78,10 @@ class ResourceMapper {
 
     // Determine the URL by chopping off everything after the dollar sign
     const pathname = this._removeDollarExtension(path)
-    const url = `${this.resolveUrl(hostname)}${encodeURI(pathname)}`
+    console.log(pathname)
+    const url = `${this.resolveUrl(hostname)}${
+      pathname.split('/').map((component) => encodeURIComponent(component)).join('/')
+    }`
```

Comment:
I would split on the filesystem path separator char [`path.sep`](https://nodejs.org/docs/latest/api/path.html#path_path_sep).

Reply:
Gotcha